test/commands/platform-dependency-version.test.ts

```diff
@@ -0,0 +1,145 @@
+  })
+
+  // Test WC versions
+  describe('with `--wcVersion` param', () => {
+    describe('checks dependencies versions for a given WP & WC versions', () => {
+      test
+      .stdout()
+      .command(['platform-dependency-version', '--wpVersion=6.0.3', '--wcVersion=7.0.1', '--dependenciesJSON=test/mocks/.externalized.json'])
+      .it('renders it in a table', ctx => {
+      // Header contains the WC version.
+        expect(ctx.stdout).to.have.string('WooCommerce 7.0.1')
+
+        // Prints packages with their versions.
+        expect(ctx.stdout).to.match(/@wordpress\/data\s*6\.6\.1/ms)
+        expect(ctx.stdout).to.match(/react\s*17\.0\.2/ms)
+        expect(ctx.stdout).to.match(/foo\s*$/ms) // No version for unmatched package
+        // Prints WC versions
+        expect(ctx.stdout).to.match(/@woocommerce\/components\s*10\.3\.0/)
+        expect(ctx.stdout).to.match(/@woocommerce\/settings\s*unknown/) // Not found WC package.
+      })
+
+      test
+      .stderr()
+      .command(['platform-dependency-version', '--wpVersion=6.0.3', '--wcVersion=7.0.1', '--dependenciesJSON=test/mocks/.externalized.json'])
+      .it('warns about not found `@woocommerce/*` packages', ctx => {
+        expect(ctx.stderr).to.have.string('Warning: @woocommerce/settings version not found')
+        expect(ctx.stderr).to.have.string('WooCommerce 7.0.1')
+      })
+
+      test
+      .stdout()
+      .command(['platform-dependency-version', '--wpVersion=6.0.3', '--wcVersion=7.0.1', '--json', '--dependenciesJSON=test/mocks/.externalized.json'])
+      .it('with `--json`, returns it in JSON format', ctx => {
+        const result = ctx.returned as string[]
+        expect(result).to.deep.equal(
+          [
+            {name: 'react', wpVersion: '17.0.2', wcVersion: null, localVersion: null},
+            {name: '@wordpress/data', wpVersion: '6.6.1', wcVersion: null, localVersion: null},
+            {name: 'foo', wpVersion: null, wcVersion: null, localVersion: null},
+            {name: '@woocommerce/components', wpVersion: null, wcVersion: '10.3.0', localVersion: null},
+            {name: '@woocommerce/settings', wpVersion: null, wcVersion: 'unknown', localVersion: null},
+          ],
+        )
+      })
+    })
+
+    describe('with `--wcDEWP` param, uses a specific list of packages externalized from WC', () => {
+      test
+      .stdout()
+      .command(['platform-dependency-version', '--wpVersion=6.0.3', '--wcVersion=7.8.1', '@woocommerce/admin-layout', '--wcDEWP=7.8.1'])
+      .it('which may add some packages', ctx => {
+        // Prints packages with their versions.
+        expect(ctx.stdout).to.match(/@woocommerce\/admin-layout\s*1\.0\.0-beta\.0/ms)
+      })
+
+      test
+      .stdout()
+      .command(['platform-dependency-version', '--wpVersion=6.0.3', '--wcVersion=7.8.1', '@woocommerce/admin-layout', '--wcDEWP=7.0.1'])
+      .it('which may miss some packages', ctx => {
+        // Prints packages with their versions.
```

Comment:
This comment doesn't match the test.

Reply:
Addressed in https://github.com/woocommerce/dewped/pull/5/commits/ec1a8cdf26905cb1e4cd84da87ca96521b932a3a